src/glmfit.jl

```diff
@@ -579,6 +589,30 @@ fit(::Type{M},
     l::Link=canonicallink(d); kwargs...) where {M<:AbstractGLM} =
         fit(M, float(X), float(y), d, l; kwargs...)
 
+function fit(::Type{M},
+             f::FormulaTerm,
+             data,
+             d::UnivariateDistribution,
+             l::Link=canonicallink(d);
+             offset::Union{AbstractVector, Nothing} = nothing,
+             wts::Union{AbstractVector, Nothing} = nothing,
+             dofit::Bool = true,
+             contrasts::AbstractDict{Symbol}=Dict{Symbol,Any}(),
+             fitargs...) where {M<:AbstractGLM}
+    f, (y, X) = modelframe(f, data, contrasts)
```

Comment:
@kleinschmidt should we do explicit `apply_schema`, etc. here instead of `modelframe` convenience? I'm thinking ahead about e.g. a hypothetical future `apply_context` stage.

Reply:
oh nevermind, `modelframe` is defined below to do exactly that

Reply:
I came here to say the same thing before I looked further down 😂